cocos/platform/apple/CCCanvasRenderingContext2D-apple.mm

```diff
@@ -301,21 +301,13 @@ -(NSPoint) convertDrawPoint:(NSPoint) point text:(NSString*) text {
         point.y += _fontSize / 2.0f;
     }
 
+    // Since the web platform cannot get the baseline of the font, a special calculation is performed.
```

Comment:
a special calculation is performed. -> an additive offset is performed for all platforms.